backend/spec/requests/comments_spec.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+RSpec.describe 'CommentsController' do
+  describe 'POST /memos/:memo_id/comments' do
+    context 'コンテンツが有効な場合' do
+      let(:memo) { create(:memo) }
+      let(:valid_comment_params) { { content: Faker::Lorem.paragraph(sentence_count: 3) } }
+
+      it 'コメントが追加され、204になる' do
+        aggregate_failures do
+          expect do
+            post "/memos/#{memo.id}/comments", params: { comment: valid_comment_params }, as: :json
+          end.to change(memo.comments, :count).by(1)
```

Comment:
IMO/Idiomatic

Commentモデルのカウントが変わったことを確認するようにしましょう！
```suggestion
          end.to change(Comment, :count).by(1)
```